<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert_main</class>
 <widget class="QWidget" name="insert_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins Medium</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1931</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E5E5E5;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>0</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="insert_mainNavSearch">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #E5E5E5;
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insert_mainCreate_2">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #E5E5E5;
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Create an Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insert_mainNavInsert">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #D1D1D1;
border-radius: 5px;
height: 30px;
width: 120px;</string>
        </property>
        <property name="text">
         <string>Insert a Word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insert_mainLogout">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #E5E5E5;
border-radius: 5px;</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>650</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E5E5E5;
border-radius: 30px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: center;</string>
     </property>
     <property name="text">
      <string>Insert a Word</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insert_mainInsertButton">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>460</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #A26BA2, stop:1 #6D5EC5);
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>191</width>
       <height>66</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="insert_mainLineEdit">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #FFFFFF;
border-radius: 5px;
height:35px;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>551</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Poppins Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Meaning(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="insert_mainTextEdit">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #FFFFFF;
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="insert_mainErrorLabel">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>430</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(197, 0, 0)</string>
     </property>
     <property name="text">
      <string>Invalid Input</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
